x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -401,6 +385,50 @@ const ProcessEventDotComponents = React.memo(
       });
     }, [dispatch, nodeId]);
 
+    const handleRelatedEventRequest = useCallback(() => {
+      dispatch({
+        type: 'userRequestedRelatedEventData',
+        payload: selfId,
+      });
+    }, [dispatch, selfId]);
+
+    const handleRelatedAlertsRequest = useCallback(() => {
+      dispatch({
+        type: 'userSelectedRelatedAlerts',
+        payload: event,
+      });
+    }, [dispatch, event]);
+
+    const history = useHistory();
+    const urlSearch = history.location.search;
+
+    /**
+     * This updates the breadcrumb nav, the table view
+     */
+    const pushToQueryParams = useCallback(
```

Comment:
This is already defined in `panel.tsx`. We should schedule a work item to refactor this. Suggestion for improvement:
* define a `pushToQueryParams` hook
* don't pass `pushToQueryParams` as a prop anywhere.